test/rmath.jl

```diff
@@ -185,8 +185,8 @@ rmathcomp_tests("norm", [
     ((0.0, 1.0), -6.0:0.01:6.0),
     ((2.0, 1.0), -3.0:0.01:7.0),
     ((0.0, 0.5), -3.0:0.01:3.0),
-    ((0, 1), -3:3),
-    ((0, 1), -Float16(3):Float16(0.01):Float16(3))
```

Comment:
As I remember it, the reason was that we decided that Rmath should only deal with `Float64` and not automatically convert inputs. One reason was the conversions should be handled explicitly such that we didn't end up promoting `Float32` to `Float64` by accident but restricting some of the Rmath signatures also made it easier to handle dispatch to generic implementations which was the main purpose of this PR.

I just checked and the `Float16` tests were not really that useful since in 0.5 multiplication with `2` converted them to `Float32`s so we don't really handle `Float16`s right now. Adding the tests back would require that we more seriously handle `Float16` inputs but that is work for another day.